backends/dpdk/run-dpdk-ptf-test.py

```diff
@@ -72,15 +72,17 @@
     help="The log level to choose.",
 )
 
-# 9559 is the default P4Runtime API server port
-GRPC_PORT: int = 9559
+# 28000 is the default P4Runtime API server port
+GRPC_PORT: int = 28000
```

Comment:
I have no personal issue if you want to use a non-default port number for the P4Runtime API connection, but this change to the comment is definitely incorrect.  9559 is the default port, as mentioned in the P4Runtime API spec.